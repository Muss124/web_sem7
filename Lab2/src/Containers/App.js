import React from 'react';
import { Provider } from 'react-redux';
import store from "../store";
import { MainWeather } from "../Components/MainWeather/js/MainWeather";
import Favorite from "../Components/Favorite/js/Favorite"


export const App = (props) => {
  return (
    <div>
      <div className="app">
        <h2>Weather here</h2>
        <MainWeather name="MainWeather" />
      </div>
      <Provider store={store}>
        <Favorite name="Favorite" />
      </Provider>
    </div>
  );
}
