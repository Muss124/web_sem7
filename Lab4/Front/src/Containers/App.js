import React from 'react';
import { Provider } from 'react-redux';
import store from "../store";
import { MainWeather } from "../Components/MainWeather.js"
import Favorite from "../Components/Favorite.js"


export const App = (props) => {
  return (
    <div>
      <div className="app">
        <h2>Weather here</h2>
        <MainWeather name="MainWeather" />
      </div>
      <Provider store={store}>
        <Favorite name="Favorite" />
      </Provider>
    </div>
  );
}
