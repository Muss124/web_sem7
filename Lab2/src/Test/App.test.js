import React from 'react';
import { App } from '../Containers/App'
import { shallow } from '../enzyme';
import { shallowToJson } from 'enzyme-to-json';

describe('Testing render App', function () {
    it("Correct render", () => {
        const tree = shallow(<App />);
        expect(shallowToJson(tree)).toMatchSnapshot();
    });
});
