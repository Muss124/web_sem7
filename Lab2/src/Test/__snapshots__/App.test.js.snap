// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing render App Correct render 1`] = `
<div>
  <div
    className="app"
  >
    <h2>
      Weather here
    </h2>
    <MainWeather
      name="MainWeather"
    />
  </div>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <Connect(Favorite)
      name="Favorite"
    />
  </Provider>
</div>
`;
